messaging/src/main/java/org/axonframework/eventhandling/SimpleEventHandlerInvoker.java

```diff
@@ -102,6 +106,10 @@ public static Builder builder() {
 
     @Override
     public void handle(EventMessage<?> message, Segment segment) throws Exception {
+        logger.trace("Received event {}. Segment Id {}. Thread {}.",
```

Comment:
Please add `logger.isTraceEnabled()` to prevent invocation of getters when log level is too low.